core/src/test/java/com/linecorp/armeria/server/http/HttpServerTest.java

```diff
@@ -154,19 +154,19 @@ protected void configure(ServerBuilder sb) throws Exception {
             SelfSignedCertificate ssc = new SelfSignedCertificate();
             sb.sslContext(HTTPS, ssc.certificate(), ssc.privateKey());
 
-            sb.serviceUnder("/delay", new AbstractHttpService() {
+            sb.serviceAt("/delay/{delay}", new AbstractHttpService() {
                 @Override
                 protected void doGet(ServiceRequestContext ctx, HttpRequest req, HttpResponseWriter res) {
-                    final long delayMillis = Long.parseLong(ctx.mappedPath().substring(1));
+                    final long delayMillis = Long.parseLong(ctx.pathParam("delay"));
```

Comment:
I think it's not necessary for now. A user may use the current `DynamicHttpService` for this kind of scenario, and may get the value as a parameter. So we can think about this again at the part 2.